opencga-app/app/scripts/azure/arm/appgw/appgw.json

```diff
@@ -0,0 +1,458 @@
+							"password": "MySecretP@ss"
```

Comment:
??

Reply:
As discussed in the https://github.com/opencb/opencga/issues/1038#issuecomment-465939699 we need a placeholder SSL certificate for AppGW to be successfully setup. This self-signed (bogus) SSL certificate is required to do so. Since every pfx certificate requires a password, the password field is also required but not used when the certificate is updated with the actual certificate (so therefore rather trivial)